test/blackbox-tests/test-cases/watching/helpers.sh

```diff
@@ -22,7 +21,8 @@ with_timeout () {
 
 stop_dune () {
     with_timeout dune shutdown;
-    wait $DUNE_PID;
+    # wait for *any* child process
```

Comment:
Is this comment correct? I think this is waiting for *all* rather than *any* child processes.

Reply:
We should also just restore the old behavior outside of linux. E.g. check `uname -s` and dispatch on that.

Reply:
Done.